tests/test_controlbyweb_web_relay.py

```diff
@@ -87,6 +88,32 @@ def test_get_state_from_config(self):
         self.assertTrue(states["noise diode path enabled"])
         self.assertTrue(states["noise on"])
 
+    def test_get_analog_input(self):
```

Comment:
Recommend adding similar tests for digital inputs and 1-Wire sensors

Reply:
added tests for both.